cmd/root.go

```diff
@@ -222,14 +272,47 @@ func run(cmd *Command) error {
 		}
 	}()
 
-	// Read tool file contents
-	buf, err := os.ReadFile(cmd.tools_file)
-	if err != nil {
-		errMsg := fmt.Errorf("unable to read tool file at %q: %w", cmd.tools_file, err)
+	var yamlFileContent []byte
+
+	if cmd.prebuiltSourceType != "" && cmd.tools_file != "" {
+		errMsg := fmt.Errorf("--prebuilt and --tools-file flags cannot be used simultaneously")
 		cmd.logger.ErrorContext(ctx, errMsg.Error())
 		return errMsg
 	}
-	toolsFile, err := parseToolsFile(ctx, buf)
+
+	if cmd.prebuiltSourceType != "" {
+		content, ok := prebuiltToolYAMLs[cmd.prebuiltSourceType]
+		if !ok {
+			var availablePrebuiltKeys []string
+			for k := range prebuiltToolYAMLs {
+				availablePrebuiltKeys = append(availablePrebuiltKeys, k)
+			}
+			prebuiltHelpSuffix := "No prebuilt configurations found."
+			if len(availablePrebuiltKeys) > 0 {
+				prebuiltHelpSuffix = fmt.Sprintf("Available: %s", strings.Join(availablePrebuiltKeys, ", "))
+			}
+			errMsg := fmt.Errorf("prebuilt source type '%s' not found. %s", cmd.prebuiltSourceType, prebuiltHelpSuffix)
+			cmd.logger.ErrorContext(ctx, errMsg.Error())
+			return errMsg
+		}
+		yamlFileContent = content
+		cmd.logger.InfoContext(ctx, "Using prebuilt tool configuration -", "source_type", cmd.prebuiltSourceType)
+	} else {
+		// Set default value of tools-file flag to tools.yaml
+		if cmd.tools_file == "" {
+			cmd.tools_file = "tools.yaml"
+		}
+		// Read tool file contents
+		buf, err := os.ReadFile(cmd.tools_file)
+		if err != nil {
+			errMsg := fmt.Errorf("unable to read tool file at %q: %w", cmd.tools_file, err)
+			cmd.logger.ErrorContext(ctx, errMsg.Error())
+			return errMsg
+		}
+		yamlFileContent = buf
```

Comment:
nit: this is kind of the 'default' use case -- should we push it first to represent that?